installer/PowerToysSetupCustomActions/PowerToysSetupCustomActions.vcxproj

```diff
@@ -90,6 +90,10 @@
       <TargetMachine>MachineX64</TargetMachine>
       <UACExecutionLevel>HighestAvailable</UACExecutionLevel>
     </Link>
+    <CustomBuildStep Label="BuildMSIX">
+      <Command>cd $(SolutionDir)MSIX &amp;&amp; makeappx build /overwrite /f PackagingLayout.xml /id x64 /op bin\ &amp;&amp; signtool sign /a /fd SHA256 /f PowerToys_TemporaryKey.pfx /p 12345 bin\x64.msix</Command>
+      <Outputs>$(SolutionDir)MSIX\bin\x64.msix</Outputs>
```

Comment:
It's pretty cool that we can create both the MSI and the MSIX with one solution, but we are going to deprecate the MSI installer so we should have the MSIX build command by itself.
Also rename `x64.msix` to `PowerToys-x64.msix` or `PowerToysPackage-x64.msix`

Reply:
shouldn't the MSIX itself contain all 3 architectures versus a targetted 64bit one?

Reply:
We can call it `PowerToys.msix` (not sure if there is a convention for the package name, we should look into it).

Reply:
@crutkas it's one of the build options, yes, and is called a "bundle". we don't have explicit x86/arm support though.

Reply:
we will have those in the future.  The main MSIX we'd ship would i would imagine would host all 3 architectures as I believe that is what the store requires as well.  A single file.  correct?

Reply:
@crutkas Looks like having a bundle [isn't required](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#create-an-app-package-upload-file).